### DISCUSSION TITLE
这个软件默认什么编码

### DISCUSSION THREAD

**Discussion by yuyi2439** [howmanybots/onebot#101] [问答] [answered]
这个软件默认什么编码
我用的是post访问

**Comment by stdrc** [comment, marked as answer]
这不是一个软件，这是一个接口标准，标准的实现才是软件。

接口标准 [规定](https://github.com/howmanybots/onebot/blob/master/v11/specs/communication/README.md) 通信使用 UTF-8 编码，所以一个正确的实现应该是 UTF-8 编码。
